.github/workflows/ci.yml

```diff
@@ -29,8 +29,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: ['5.6', '7.0', '7.1', '7.2', '7.3', '7.4']
-        wordpress: ['5.9']
+        include:
+          - php: '5.6'
+            composer: 1
+            wordpress: '5.9'
+          - php: '7.0'
+            composer: 1
+            wordpress: '5.9'
+          - php: '7.1'
+            composer: 1
+            wordpress: '5.9'
+          - php: '7.2'
+            composer: 2
+            wordpress: '5.9'
+          - php: '7.3'
+            composer: 2
+            wordpress: '5.9'
+          - php: '7.4'
+            composer: 2
+            wordpress: '5.9'
```

Comment:
I found two ways to make this change work. 

The first is as you can see here using the `includes` syntax for [matrices](https://docs.github.com/en/actions/using-jobs/using-a-build-matrix-for-your-jobs#using-environment-variables-in-a-matrix), which I believe is likely the best option as it means you can combine specific versions which might be necessary later on too.

Alternatively I initially tried this more simple method: https://github.com/accredible/accredible-learndash-add-on/blob/137a44d96a0ebe309865ec495d233741e240dbab/.github/workflows/ci.yml#L47-L61 which also worked ([logs here](https://github.com/accredible/accredible-learndash-add-on/runs/5786229076?check_suite_focus=true))

Just highlighting in case there might be some reason I'm not aware of to avoid this approach!

Reply:
Thanks for sharing the alternative. I like the current approach better too considering that we'll want to combine different versions in the future.